sdk/contentsafety/azure-ai-contentsafety/README.md

```diff
@@ -242,78 +265,176 @@ def create_or_update_text_blocklist():
             raise
         print(e)
         raise
+```
+
+<!-- END SNIPPET -->
+
+#### List text blocklists
+<!-- SNIPPET:sample_manage_blocklist.list_text_blocklists -->
+
+```python
+
+    import os
+    from azure.ai.contentsafety import ContentSafetyClient
+    from azure.core.credentials import AzureKeyCredential
+    from azure.core.exceptions import HttpResponseError
+
+    key = os.environ["CONTENT_SAFETY_KEY"]
+    endpoint = os.environ["CONTENT_SAFETY_ENDPOINT"]
+
+    # Create an Content Safety client
+    client = ContentSafetyClient(endpoint, AzureKeyCredential(key))
 
-def add_block_items():
-    block_items = [TextBlockItemInfo(text=block_item_text_1), TextBlockItemInfo(text=block_item_text_2)]
     try:
-        result = client.add_block_items(
-            blocklist_name=blocklist_name,
-            body=AddBlockItemsOptions(block_items=block_items),
-        )
-        if result is not None and result.value is not None:
-            print("\nBlock items added: ")
-            for block_item in result.value:
-                print("BlockItemId: {}, Text: {}, Description: {}".format(block_item.block_item_id, block_item.text, block_item.description))
+        blocklists = client.list_text_blocklists()
+        if blocklists is not None:
```

Comment:
Really, stop testing everything against None :). This operation will never return `None`, this is misleading for customers :)

Reply:
See above reply